File: app/src/main/java/com/jhj/rongim/TResult.java
package com.jhj.rongim;

import com.jhj.httplibrary.result.Result;

/**
 * Created by jhj on 18-6-9.
 */

public class TResult<T> extends Result {

    private T items;
    private int code;

    public T getItems() {
        return items;
    }

    public void setItems(T items) {
        this.items = items;
    }
}

File: app/src/main/java/com/jhj/rongim/net/TResult.kt
package com.jhj.rongim.net

import com.jhj.httplibrary.result.Result

/**
 * Created by jhj on 18-6-9.
 */

class TResult<T> : Result<T>() {

    override val clazz: Result<T>
        get() = this


    var code: Int = 0
    var userId: String? = null
    val token: T? = null



}
